ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -199,6 +200,28 @@ func convertToContainerDef(inputCfg *adapter.ContainerConfig, ecsContainerDef *C
 	return outputContDef, nil
 }
 
+// ECS Params healthcheck overrides the healthcheck from docker compose
+func overrideHealthCheck(outputContDef *ecs.ContainerDefinition, ecsParamsHealth ecs.HealthCheck) {
+	if outputContDef.HealthCheck == nil {
+		outputContDef.SetHealthCheck(&ecsParamsHealth)
+	}
+	if len(ecsParamsHealth.Command) > 0 {
```

Comment:
Should we have an error if command is not present, since it's the only required field on HealthCheck?